website/src/theme/Footer/index.js

```diff
@@ -0,0 +1,110 @@
+/**
```

Comment:
Just use the default Footer, don't need to swizzle.

Reply:
unswizzling here just means to delete the Footer file right?